### DISCUSSION TITLE
Change the compression type of a table with dynamic partitioning enabled to zstd

### DISCUSSION THREAD

**Discussion by fdf1779** [apache/doris#33066] [A - General / Q&A]
What should I do to change a dynamically partitioned table using the default compression format to zstd compression without rebuilding the table? Can I directly use the Alter Table example_db. my_table set ("compression"="zstd")? Will the modification affect the historical partitioning

**Comment by yagagagaga** [comment]
You can't do it if you don't want to rebuild table.
